crates/compression/src/compressed_block_payload/v1.rs

```diff
@@ -29,12 +32,17 @@ pub struct CompressedBlockHeader {
     pub application: ApplicationHeader<Empty>,
     /// The consensus header.
     pub consensus: ConsensusHeader<Empty>,
-    // The block id.
+    /// The block id.
     pub block_id: BlockId,
+    /// The registry root after execution of this particular block.
+    pub registry_root: Bytes32,
```

Comment:
I might miss something because I don't have the full context but why on the other files we are placing this under a feature `fault-prooving` and here not ?

Reply:
this module is only imported when that feature flag is enabled :)